packages/translations/src/clientKeys.ts

```diff
@@ -333,14 +333,24 @@ export const clientTranslationKeys = createClientTranslationKeys([
   'upload:fileSize',
 
   'validation:emailAddress',
+  'validation:enterNumber',
   'validation:fieldHasNo',
+  'validation:greaterThanMax',
+  'validation:invalidInput',
+  'validation:invalidSelection',
+  'validation:invalidSelections',
+  'validation:lessThanMin',
   'validation:limitReached',
   'validation:longerThanMin',
+  'validation:notValidDate',
   'validation:required',
   'validation:requiresAtLeast',
+  'validation:requiresNoMoreThan',
+  'validation:requiresTwoNumbers',
   'validation:shorterThanMax',
-  'validation:greaterThanMax',
+  'validation:trueOrFalse',
```

Comment:
Looks like greaterThanMax was removed but it's used in various places [like this](https://github.com/payloadcms/payload/blob/main/packages/payload/src/fields/validations.ts#L793)